index.bs

```diff
@@ -1,12 +1,12 @@
 <pre class='metadata'>
-Title: The TODO API
-Shortname: todo
+Title: The WebRequest.SecurityInfo API
```

Comment:
Remove the dot. This is more about style and title, as opposed to nesting/namespace/syntax.